styleguide/_themes/global/scss/_grid.scss

```diff
@@ -0,0 +1,22 @@
+@mixin make-container-fluid {
+  @include container-fixed;
+}
+
+@mixin make-container {
+  @include make-container-fluid;
+  max-width: 960px;
+
+  @media (min-width: $screen-sm-min) {
+    width: $container-sm;
+  }
+  @media (min-width: $screen-md-min) {
+    width: $container-md;
+  }
+  @media (min-width: $screen-lg-min) {
+    width: $container-lg;
+  }
+}
+
+.row {
```

Comment:
`.row` already exists in Bootstrap in this exact fashion:

https://github.com/twbs/bootstrap/blob/10606a73bcbfc6c723d60e2a1069c921afff0d2a/less/grid.less#L39-L41
https://github.com/twbs/bootstrap/blob/daad359f016949725f4ba39645e3f9ade7dda246/dist/css/bootstrap.css#L1612

In the spirit of MaintainableCSS having the properties duplicated is not a bad thing and provides better defense against regression; however, that is somewhat lessened when mixins are involved as anything including it could be affected if there were a change.

If we are dumping the columns in favor of using mixins then dumping a row in favor of a mixin seems fine too, no? I agree that a placeholder class would be well-suited so we get the best of both worlds.